README.md

```diff
@@ -260,11 +260,11 @@ root: ~/<%= @settings["workspace"] %>
 This will fire up tmux with all the tabs and panes you configured.
 
 ```
-tmuxinator start [project] [alias]
+tmuxinator start [project] -n [name]
 ```
 
-If you use the optional `[alias]` argument, it will start a new tmux session
-with the custom alias name provided.  This is to enable reuse of a project
+If you use the optional `[name]` argument, it will start a new tmux session
+with the custom name name provided.  This is to enable reuse of a project
```

Comment:
This change looks good! My only commentary is to just say "custom name provided" instead of "custom name name provided".